File: Example/ZBarObjC/ZBarObjC/ViewController.m
//
//  ViewController.m
//  ZBarObjC
//
//  Created by Francesco Frison on 09/08/2014.
//  Copyright (c) 2014 Ziofritz. All rights reserved.
//

#import "ViewController.h"
#import "ZBarSDK.h"

@interface ViewController ()

@property (nonatomic, strong) ZBarReaderView *readerView;

@end

@implementation ViewController
            
- (void)viewDidLoad {
    [super viewDidLoad];
    ZBarImageScanner *scanner = [[ZBarImageScanner alloc] init];
    self.readerView = [[ZBarReaderView alloc] initWithImageScanner:scanner];
    
    [self.view addSubview:self.readerView];
    
    self.readerView.frame = self.view.bounds;
    [self.readerView start];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Example/ZBarSwift/ZBarSwift/ViewController.swift
//
//  ViewController.swift
//  ZBarSwift
//
//  Created by Francesco Frison on 09/08/2014.
//  Copyright (c) 2014 Ziofritz. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    var readerView : ZBarReaderView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.readerView = ZBarReaderView(imageScanner: ZBarImageScanner())
        self.view.addSubview(self.readerView!)
        self.readerView!.start()
        
        self.readerView!.frame = self.view.bounds
        
        if UIDevice.currentDevice().model == "iPhone Simulator" {
            println("Test Device Only")
        }
        
    }

}

